attorch/regularizers.py

```diff
@@ -24,6 +25,25 @@ def laplace3d():
     return l.astype(np.float32)[None, None, ...]
 
 
+#def fft_smooth(grad, factor=1/4):
```

Comment:
same code as in staticnet_analyses/utils.py, we could drop it from here

Reply:
Good point - let me clean it up